roachtest: gorm failed

roachtest.gorm [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4414364&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4414364&tab=artifacts#/gorm) on master @ [8d0a1773a6b55f06c04ea77e6a39426e32ba9a63](https://github.com/cockroachdb/cockroach/commits/8d0a1773a6b55f06c04ea77e6a39426e32ba9a63):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/gorm/run_1
	orm_helpers.go:190,orm_helpers.go:116,java_helpers.go:220,gorm.go:127,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1469-g8d0a1773a6
		Tests run against gorm v1.22.5
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_1) is available in the artifacts' gorm log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4417791&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4417791&tab=artifacts#/gorm) on master @ [ae6d0339ced65e63f2baca0ace8faf9ad171bbf6](https://github.com/cockroachdb/cockroach/commits/ae6d0339ced65e63f2baca0ace8faf9ad171bbf6):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/gorm/run_1
	orm_helpers.go:190,orm_helpers.go:116,java_helpers.go:220,gorm.go:127,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1477-gae6d0339ce
		Tests run against gorm v1.23.1
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_1) is available in the artifacts' gorm log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4421693&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4421693&tab=artifacts#/gorm) on master @ [1a89121d67fa02d5c674766ea01d3c78455fd28b](https://github.com/cockroachdb/cockroach/commits/1a89121d67fa02d5c674766ea01d3c78455fd28b):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/gorm/run_1
	orm_helpers.go:190,orm_helpers.go:116,java_helpers.go:220,gorm.go:127,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1485-g1a89121d67
		Tests run against gorm v1.23.1
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_1) is available in the artifacts' gorm log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #76823 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4427196&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4427196&tab=artifacts#/gorm) on master @ [4c65a10fea9762b97559c1a488cd5b96cb2cf81a](https://github.com/cockroachdb/cockroach/commits/4c65a10fea9762b97559c1a488cd5b96cb2cf81a):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/gorm/run_1
	orm_helpers.go:190,orm_helpers.go:116,java_helpers.go:220,gorm.go:127,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1511-g4c65a10fea
		Tests run against gorm v1.23.1
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_1) is available in the artifacts' gorm log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #76823 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4436786&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4436786&tab=artifacts#/gorm) on master @ [1d13605e2aeb1af0a5536f1f460e45d557119128](https://github.com/cockroachdb/cockroach/commits/1d13605e2aeb1af0a5536f1f460e45d557119128):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/gorm/run_1
	orm_helpers.go:190,orm_helpers.go:116,java_helpers.go:220,gorm.go:127,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1561-g1d13605e2a
		Tests run against gorm v1.23.1
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_1) is available in the artifacts' gorm log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #76823 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/gorm.go
+++ b/pkg/cmd/roachtest/tests/gorm.go
@@ -24,7 +24,7 @@ import (
 )
 
 var gormReleaseTag = regexp.MustCompile(`^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)$`)
-var gormSupportedTag = "v1.22.2"
+var gormSupportedTag = "v1.23.1"
 
 func registerGORM(r registry.Registry) {
 	runGORM := func(ctx context.Context, t test.Test, c cluster.Cluster) {

